src/servers/src/http/prometheus.rs

```diff
@@ -398,6 +401,10 @@ async fn get_all_column_names(
         }
     }
 
+    let _ = labels.insert(METRIC_NAME.to_string());
+    let _ = labels.remove(GREPTIME_TIMESTAMP);
```

Comment:
BTW, all field columns SHOULD BE filtered from the response, and the GREPTIME_VALUE is just a special field column